### ISSUE TITLE
First + from an Telefonlink is deleted

### ISSUE THREAD

**Issue by Newan** [zammad/zammad#2018]
### Infos:

* Used Zammad version: 2.4
* Installation method (source, package, ..): package deb
* Operating system: ubuntu 16.04
* Database + version: postgresql
* Elasticsearch version: n/a
* Browser + version: Chrome Version 66.0.3359.170 

### Expected behavior:

* not delete the first + character for country calls like +49 and create the telefonlink in zammad with + an front like tel:+4911111111

### Actual behavior:

* the first + character deleted from a telephone number for country calls like +49
the telephone link is missing the country character -> tel:4911111111

curl -X POST --data "event=newCall&from=+492111234567&to=+4915791234567&direction=in&callId=1234561&user[]=Alice&user[]=Bob"

![image](https://user-images.githubusercontent.com/3693094/39997523-7bfc3808-5783-11e8-9f95-4662f11e35e4.png)

### Steps to reproduce the behavior:

* make an sipgate call with +49 oder %2b49 

Yes I'm sure this is a bug and no feature request or a general question.

**Comment by martini** [comment]
@Newan 

I just looked into my sipgate.io log, there is no + in my log. 

For example:
```  
event=newCall&direction=in&from=4912341234&to=49123412345&callId=54566B150B0C0D&user%5B%5D=Denny+Smith&user%5B%5D=Johanna+Smith&user%5B%5D=Johannes+Smith&user%5B%5D=Martin+Smith&user%5B%5D=Natascha+Smith&user%5B%5D=Roy+Smith&user%5B%5D=Thorsten+Smith&userId%5B%5D=w8&userId%5B%5D=w10&userId%5B%5D=w2&userId%5B%5D=w0&userId%5B%5D=w6&userId%5B%5D=w17&userId%5B%5D=w4&fullUserId%5B%5D=1452406w8&fullUserId%5B%5D=1452406w10&fullUserId%5B%5D=1452406w2&fullUserId%5B%5D=1452406w0&fullUserId%5B%5D=1452406w6&fullUserId%5B%5D=1452406w17&fullUserId%5B%5D=1452406w4&diversion=1452406g6
```

So do you really get the leading + pushed by sipgate?

 -Martin

**Comment by Newan** [comment]
No we doesn´t use sipgate. i wrote my own extension in our swyxIT!. (i can published this?)
 if you send this event with an leading +, this + is removed. but the swyit required this in the telefonlink to used this on clicktocall 

on https://github.com/zammad/zammad/issues/991 hiaz explained : "(Just to be sure) Throw away every + which is not the first sign", like the RFC

**Comment by rlue** [comment]
Fixed by 312278c2.
